### DISCUSSION TITLE
第6节课作业(12班)

### DISCUSSION THREAD

**Discussion by fanqiNO1** [InternLM/Tutorial#465] [作业]
xdm，最后一次作业了，加油！

**Comment by HuangXinzhe** [comment]
https://blog.csdn.net/qq_29787929/article/details/135758447

**Comment by jujimeizuo** [comment]
https://note.jujimeizuo.cn/llm/internlm/lec6/

**Comment by cheng221** [comment]
https://spice-spice-829.notion.site/9dd7c576c48b44d4b2347844513cbfbc?pvs=4

**Comment by canxingliuying** [comment]
https://blog.csdn.net/weixin_42449385/article/details/135796992?spm=1001.2014.3001.5502

**Comment by ChengaoJ** [comment]
https://blog.csdn.net/weixin_43193528/article/details/135831583

**Comment by sunhanyu714** [comment]
https://github.com/sunhanyu714/Internlm_learning/blob/main/%E7%AC%AC%E5%85%AD%E8%8A%82%E8%AF%BE%E4%BD%9C%E4%B8%9A

**Comment by 0-yy-0** [comment]
https://blog.csdn.net/qq_34354951/article/details/135834119

**Comment by deseijily** [comment]
https://j1b8l2ps5tm.feishu.cn/docx/AKIEdCMbhoHETBxbnkjcfTiInEf

**Comment by ltstar** [comment]
[课程6-OpenCompass.md](https://github.com/InternLM/tutorial/files/14060883/6-OpenCompass.md)

**Comment by Smiling-Weeping-zhr** [comment]
https://www.cnblogs.com/smiling-weeping-zhr/p/17990602
